packages/ckeditor5-dev-env/lib/release-tools/changelog/templates/commit.hbs

```diff
@@ -57,4 +57,10 @@
     {{~this.repository}}#{{this.issue}}
   {{~/if}}{{/each}}
 {{~/if}}
+{{~#if body}}
+
```

Comment:
So please check what happens if you change it into one space. I've been wondering why there's a duplicated space before https://raw.githubusercontent.com/ckeditor/ckeditor5-engine/master/CHANGELOG.md so perhaps this is somehow related?